lms/djangoapps/courseware/views/index.py

```diff
@@ -365,6 +365,7 @@ def _create_courseware_context(self):
             # Disable student view button if user is staff and
             # course is not yet visible to students.
             courseware_context['disable_student_access'] = True
+            courseware_context['supports_preview_menu'] = False
```

Comment:
`'preview_menu_enabled'` might be a better name for both the old and new functionality.